tests/integration/test_permissions.py

```diff
@@ -57,7 +58,7 @@ def _create_and_index_record(record):
     return record
 
 
-@pytest.fixture(scope='function')
+@pytest.fixture(scope='session')
```

Comment:
That might have been related to https://github.com/inspirehep/inspire-next/pull/1719 . Once it is merged, you can test without this change and should work ok.